### PR TITLE
sets op 2 vars

### PR DESCRIPTION
**Pull request by latot**
Hi, here simplified version.
Fixes https://github.com/cbm755/octsympy/issues/147

### CODE REVIEW

File: inst/@sym/intersect.m [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+  cmd = {
+         'a, b = _ins'
+         'if isinstance(a, sp.Set) or isinstance(b, sp.Set):'
+         '    return a & b, 1'
+         ''
+         'A = sp.FiniteSet(*(list(a) if isinstance(a, sp.MatrixBase) else [a]))'
+         'B = sp.FiniteSet(*(list(b) if isinstance(b, sp.MatrixBase) else [b]))'
+         'C = A & B'
+         'return sp.Matrix([[list(C)]]), 0'
```
Comment (cbm755) [inline inst/@sym/intersect.m]: `'return sp.Matrix([list(C)]),'

(and drop the horzcat stuff)

File: inst/@sym/setdiff.m [hunk 2]
```diff
@@ -26,35 +26,24 @@
...
-  % FIXME: is it worth splitting out a "private/set_helper"?
+  cmd = {
+         'a, b = _ins'
+         'if isinstance(a, sp.Set) or isinstance(b, sp.Set):'
+         '    return a - b, 1'
+         ''
+         'A = sp.FiniteSet(*(list(a) if isinstance(a, sp.MatrixBase) else [a]))'
+         'B = sp.FiniteSet(*(list(b) if isinstance(b, sp.MatrixBase) else [b]))'
+         'C = A - B'
+         'return sp.Matrix([[list(C)]]), 0'
```
Comment (cbm755) [inline inst/@sym/setdiff.m]: same here

File: inst/@sym/setxor.m [hunk 3]
```diff
@@ -26,39 +26,32 @@
...
+    [r, out] = python_cmd (cmd, sym(a), sym(b));
+
+    if !out
+      r = horzcat(r{:});
+    end
 
 end
 
 
+
```
Comment (cbm755) [inline inst/@sym/setxor.m]: upstream octave style says exactly two blank spaces after function end and before  tests

File: inst/@sym/unique.m [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+%%
+%% @seealso{union, intersect, setdiff, setxor, ismember}
+%% @end deftypefn
+
+%% Author: Colin B. Macdonald
+%% Keywords: symbolic
+
+function r = unique(A)
+
+  S = python_cmd ('return list(uniq(*_ins)),', A);
```
Comment (cbm755) [inline inst/@sym/unique.m]: If input is a FiniteSet, maybe we don't want to the list?  Does it work with just `uniq()` for now?

Comment (latot) [inline inst/@sym/unique.m]: uniq return without list:
```
pydebug: error exporting variable:
pydebug: x: <generator object uniq at 0x7f84ab0d8be0>
pydebug: type: <type 'generator'>
errors ....
```

**Comment by latot** [comment]
fixed

**Comment by cbm755** [comment]
I have one more fix on the other branch, but I'll merge this and then merge my fix.

**Pull request merged**